src/test/Simulation.t.sol

```diff
@@ -516,7 +516,8 @@ contract SimulationTests is DSTest {
     }
 
     function testBarkLpDaiEth() public {
-        uint256 amount = 100 * WAD;
+        (,,,, uint256 dustRad) = vat.ilks(lpDaiEthName);
+        uint256 amount = (dustRad / (getLpDaiEthPrice() * 10 ** 9)) * 2;
```

Comment:
can you explain ? Is this amount higher than `dust / RAY` ?

Reply:
The amount used before resulted in a frob which tried to take out debt smaller than this ilk's debt (which was recently raised to 60K dai).
Hence we needed to increase it.
I changed the amount to twice the dust amount (doing the conversion from dai units to collateral units).